tests/common.go

```diff
@@ -7180,24 +7180,22 @@ func EndPxBackupTorpedoTest(contexts []*scheduler.Context) {
 		AfterEachTest(contexts, CurrentTestRailTestCaseId, RunIdForSuite)
 	}
 	ginkgoTestDescr := ginkgo.CurrentGinkgoTestDescription()
-	if ginkgoTestDescr.Failed {
```

Comment:
How about?

```go
if ginkgoTestDescr.Failed {
    log.Infof(">>>> FAILED TEST: %s", ginkgoTestDescr.FullTestText)
}
log.Infof(">>>> Collecting logs for testcase : %s", ginkgoTestDescr.FullTestText)
.
.
.
```

Reply:
we don't want to differentiate between test which passed or failed , one string "Collecting logs for testcase" should be good enough

Reply:
done retained the FAILED TEST comment